src/Network/Receive/kRO/Sakexe_0.pm

```diff
@@ -1582,6 +1583,13 @@ sub inventory_expansion_result {
 	}
 }
 
+sub received_login_token {
+	my ($self, $args) = @_;
+	my $master = $masterServers{$config{master}};
+	# rathena use 0064 not 0825
+	$messageSender->sendTokenToServer($config{'username'}, $config{'password'}, $master->{master_version}, $master->{version});
```

Comment:
add check for xkore 1/3 this could cause duplicated packet

Reply:
fixed : a4297ae